bigtable/tests/admin_integration_test.cc

```diff
@@ -257,13 +319,11 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
-  std::string const project_id = argv[0];
-  std::string const instance_id = argv[1];
-
+  std::string const project_id = argv[1];
```

Comment:
Wow, this was really broken but since we test against the emulator we never noticed.  Thanks!